spec/models/playbook_gear_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PlaybookGear, type: :model do
+  let(:gear) { create :gear }
+  let(:playbook) { create :playbook }
+
+  it 'correctly pulls gear from a playbook' do
+    playbook.gears << gear
+    expect(playbook.gears.ids).to include(gear.id)
```

Comment:
Rails has a helper you can use here:
```suggestion
    expect(playbook.gear_ids).to include(gear.id)
```
Totally optional, but it's a nice piece of syntactic sugar.

Reply:
Poured some sugar on it.  Fixed with commit 3c872f3.